.gitignore

```diff
@@ -1,4 +1,14 @@
 dist/
 node_modules/
-.vscode/
-npm-debug.log
\ No newline at end of file
+npm-debug.log
+
+# Created by https://www.gitignore.io/api/visualstudiocode
+
+### VisualStudioCode ###
+.vscode/*
+!.vscode/settings.json
```

Comment:
Would [.editorconfig](http://editorconfig.org/) be better for that, given that it's editor agnostic?